features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -560,7 +560,13 @@
 
     <feature name="openhab-binding-netatmo" description="Netatmo Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.netatmo/${project.version}</bundle>
```

Comment:
@J-N-K Do we have to add the reference to wrap feature here too?

```
        <feature prerequisiste="true">wrap</feature>
```

Reply:
Should I submit a PR for this ?